content/help/ops/perf/index.md

```diff
@@ -0,0 +1,78 @@
+
+| Feature | Default | Description |
+| --- | --- | --- |
+| `global.proxy.accessLogFile` | `""` |  Proxy access logging is disabled because it contributes to high CPU and I/O. Set this to `/dev/stdout` to enable access logging.|
+| `mixer.adapters.stdio.enabled` | `false` |  Mixer access logging is disabled because it is a debug feature and it contributes to high CPU and I/O. Set this to `true` to enable access logging.|
+| `global.disablePolicyChecks` | `true` | Enable policy checks *only* if you are using rate limiting or another istio-policy feature.|
+
+### Namespace isolation
+
+Namespace isolation using the sidecar resource is a major performance improvement in Istio 1.1
+over the previous release of Istio. When namespace isolation is turned on-
+
+- The default connectivity is namespace wide. Previously it was mesh wide.
+- Pilot configures envoy with information that is relevant to routing within the namespace.
+- Proxy receives less configuration information, therefore the proxy memory usage is lower.
+
+The following tables documents the reduction in resource utilization due to namespace isolation for 2 different setups.
+
+Configuration of each setup:
+
+| Parameter | Setup 1 | Setup 2 |
+| --- | --- | --- |
+| Namespaces (N)| 40 | 50 |
+| Sidecars (N x 40) | 1600 | 2000 |
+| Services (N x 20) | 800 | 1000 |
+| Virtual Services (N x 2) | 80 | 100|
+| Gateways (N) | 40 | 50 |
+| Configuration change rate (N/4 per minute) | 10/min | 12/min |
+
+, and a comparison of resource utilization for each, with and without namespace isolation:
+
+| Resource | Setup 1 || Setup 2 ||
+| --- | --- | --- | --- | --- |
+| Namespace Isolation | Enabled |  Disabled | Enabled | Disabled |
+| Pilot replicas | 1  | 2 | 1 | 3 |
+| Pilot CPU      | 1 vCPU | 5 vCPU | 1 vCPU | 8 vCPU |
+| Pilot Memory   | 1.4 GB | 4 GB | 1.4 GB | 8 GB |
+| Proxy Memory (per instance) | 50 MB  | 170 MB | 50 MB  | 225 MB |
+
+### Istio Sizing and Capacity Planning Guidance
+
+Based on all the performance data available, one can have an educated sizing and capacity planning decisions for Istio. The capacity planning effort depends on the kind of application, transaction volume and what features of Istio are enabled among many other factors. It can be looked at from how much total throughput the application has to support at what latency and can also be viewed as how many requests pass through the mesh. The Istio Performance Grafana dashboard is updated to provide this data for any workload using Istio.
+
+The following table summarizes the resources requirements for Istio main components for the default configuration:
+
+| Istio Component | CPU | Memory | Condition |
+| --- | --- | --- | --- |
+| Proxy | 0.6 vCPU | 50 MB `*` | CPU: 1000 mesh requests/sec; Mem: 1000 services, 2000 sidecars |
+| Telemetry | 0.6 vCPU | 200 MB | 1000 mesh requests/sec |
+| Pilot | 1 vCPU | 1500 MB | 1000 services, 2000 sidecars |
+
```

Comment:
is this with ns isolation on or off?  by default, it is off.

Reply:
NS isolation is on. Will make a remark.